blocks/init/src/Blocks/assets/scripts/helpers/cleanObject.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Removes `null`, `undefined` and empty entries from a given object.
+ *
+ * @param {object} object - Object which may contain `null`, `undefined` or empty values.
+ *
+ * @returns {object} Object without `null`, `undefined` and empty values.
+ */
+export function cleanObject(object) {
```

Comment:
this file should go to the frontend-libs helpers folder

https://github.com/infinum/eightshift-frontend-libs/tree/main/scripts/helpers